src/configurations/nf/routesConfig.ts

```diff
@@ -41,37 +41,55 @@ const routes: GenericRoute[] = [
         },
```

Comment:
Moved a couple of components around in the nf config, to be closer to the design.